.ci/appveyor.bat

```diff
@@ -27,10 +27,7 @@ mvn -e verify -Dcheckstyle.ant.skip=true -Dcheckstyle.skip=true -Dspotbugs.skip=
 :: powermock doesn't support modifying final fields in JDK12
 if "%OPTION%" ==  "verify_without_checkstyle_JDK12" (
 mvn -e verify -Dcheckstyle.ant.skip=true -Dcheckstyle.skip=true -Dspotbugs.skip=true^
- -Dtest=!FileContentsTest#testGetJavadocBefore,!FileTextTest#testFindLine*,^
-!MainFrameModelPowerTest#testOpenFileWithUnknownParseMode,^
-!TokenUtilTest#testTokenValueIncorrect2,^
-!ImportControlLoaderPowerTest#testInputStreamThatFailsOnClose^
+ -Dtest=!ImportControlLoaderPowerTest#testInputStreamThatFailsOnClose^

```

Comment:
Remaining will be resolved in some other issue or PR ? 
Or ?

Reply:
I'll see what I can do. Not in this one in order to keep PR small.

Reply:
good.